atomic_reactor/inner.py

```diff
@@ -511,10 +512,6 @@ def build_docker_image(self):
             postbuild_runner = PostBuildPluginsRunner(self.builder.tasker, self,
                                                       self.postbuild_plugins_conf,
                                                       plugin_files=self.plugin_files)
-            exit_runner = ExitPluginsRunner(self.builder.tasker, self,
-                                            self.exit_plugins_conf,
-                                            keep_going=True,
-                                            plugin_files=self.plugin_files)
```

Comment:
The reason for creating all the runners before the pre-build phase is that their `__init__` methods get to fail early if (e.g.) a required executable is missing or whatever.

Reply:
OK, let's leave it like this.